core/src/main/java/org/infinispan/configuration/serializing/CoreConfigurationSerializer.java

```diff
@@ -570,6 +573,20 @@ private void writeMetrics(ConfigurationWriter writer, GlobalConfiguration global
       }
    }
 
+   private void writeTracing(ConfigurationWriter writer, GlobalConfiguration globalConfiguration) {
+      GlobalTracingConfiguration tracing = globalConfiguration.tracing();
+      AttributeSet attributes = tracing.attributes();
+      if (attributes.isModified()) {
+         writer.writeStartElement(Element.TRACING);
+         attributes.write(writer, GlobalTracingConfiguration.COLLECTOR_ENDPOINT, Attribute.COLLECTOR_ENDPOINT);
+         attributes.write(writer, GlobalTracingConfiguration.ENABLED, Attribute.ENABLED);
+         attributes.write(writer, GlobalTracingConfiguration.EXPORTER_PROTOCOL, Attribute.EXPORTER_PROTOCOL);
+         attributes.write(writer, GlobalTracingConfiguration.SERVICE_NAME, Attribute.SERVICE_NAME);
+         attributes.write(writer, GlobalTracingConfiguration.SECURITY, Attribute.SECURITY);
+         writer.writeEndElement();
+      }
+   }
```

Comment:
I think this can be replaced with the following:
```suggestion
      globalConfiguration.tracing().attributes().write(writer, Element.TRACING);
```

Reply:
@pruivo Just for curiosity. Can we apply this shortcut to the other sub-configurations? Other than tracing I mean.

Reply:
yes, the  `attributes().write()` was introduced at a later stage and we didn't convert the old methods.